base_menu_visibility_restriction/demo/ir_ui_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="base.module_mi" model="ir.ui.menu">
```

Comment:
This is a non-existent module, installing the module with demo-data enabled will cause an error

Reply:
@jeroenev good point. I removed it.